### PR TITLE
Search api : Refactoring

### PR DESCRIPTION
**Pull request by PradyumnaNagendra**

### CODE REVIEW

File: content-api/content-service/conf/logback.xml [hunk 1]
```diff
@@ -11,8 +11,22 @@
 		</encoder>
 	</appender>
 
+	<appender name="rollingFile" class="ch.qos.logback.core.rolling.RollingFileAppender">
+		<rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+			<fileNamePattern>/data/logs/content-log-%d{yyyy-MM-dd}.log</fileNamePattern>
```
Comment (maheshkumargangula) [inline content-api/content-service/conf/logback.xml:16]:
```suggestion
			<fileNamePattern>/data/logs/search-log-%d{yyyy-MM-dd}.log</fileNamePattern>
```

File: search-api/search-actors/src/test/resources/log4j2.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (maheshkumargangula) [inline search-api/search-actors/src/test/resources/log4j2.xml:1]: Why we still have log4j2 configuration?

File: search-api/search-core/src/test/resources/log4j2.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (maheshkumargangula) [inline search-api/search-core/src/test/resources/log4j2.xml:1]: Why we still have log4j2 configuration?

File: search-api/search-actors/src/test/resources/log4j2.xml (hunk 2 above)
Comment (PradyumnaNagendra) [reply search-api/search-actors/src/test/resources/log4j2.xml]: removed

File: content-api/content-service/conf/logback.xml (hunk 1 above)
Comment (PradyumnaNagendra) [reply content-api/content-service/conf/logback.xml]: This change was in content-api, updated the logback accordingly

**Pull request merged**